instrumentation/spring/spring-boot-autoconfigure/README.md

```diff
@@ -392,6 +392,13 @@ If an exporter is present in the classpath during runtime and a spring bean of t
 
 <!-- Slf4j Log Correlation  otel.springboot.loggers.slf4j.enabled		true   		org.slf4j.MDC -->
 
+##### Resource Properties
+
+| Feature  | Property                                         | Default Value          |
+|----------|--------------------------------------------------|------------------------|
+| Resource | otel.springboot.resource.enabled                 | `true`                 |
+|          | otel.springboot.resource.attributes.service.name | `unknown_service:java` |
+
```

Comment:
can you add doc about `spring.application.name` and about adding custom attributes?